PendingReleaseNotes.md

```diff
@@ -5,6 +5,7 @@
 ## Notable Features
 - The `fsType` default for StorageClass examples are now using XFS to bring it in line with Ceph recommendations.
 - Ceph is updated from Luminous 12.2.5 to 12.2.7.
+- Ceph OSDs will be automatically updated by the operator when there is a change to the version. See the [OSD upgrade notes](Documentation/upgrade-patch.md#object-storage-daemons-osds).
```

Comment:
this statement isn't quite true anymore, the OSDs will be automatically updated by the operator when their configuration has changed at all (not just version) and the operator is restarted.